hedera-node/configuration/mainnet/bootstrap.properties

```diff
@@ -19,3 +19,4 @@ accounts.maxNumber=20_000_000
 #Overrides that differ based on the network
 ledger.id=0x00
 contracts.chainId=295
+contracts.maxKvPairs.individual=16384000
```

Comment:
I don't think we need to change the override files. Updating the defaults is sufficient.